dm/ctl/master/import_export_config.go

```diff
@@ -0,0 +1,383 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package master
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path"
+	"sort"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"go.etcd.io/etcd/clientv3"
+
+	"github.com/pingcap/errors"
+
+	"github.com/pingcap/dm/dm/config"
+	"github.com/pingcap/dm/dm/ctl/common"
+	"github.com/pingcap/dm/dm/pb"
+	"github.com/pingcap/dm/pkg/ha"
+	"github.com/pingcap/dm/pkg/utils"
+)
+
+var (
+	taskDirname          = "tasks"
+	sourceDirname        = "sources"
+	relayWorkersFilename = "relay_workers.json"
+	yamlSuffix           = ".yaml"
+)
+
+// NewExportCfgsCmd creates a exportCfg command.
+func NewExportCfgsCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "export-configs [--dir directory]",
+		Short: "Export the configurations of sources and tasks.",
+		RunE:  exportCfgsFunc,
+	}
+	cmd.Flags().StringP("dir", "d", "configs", "specify the destinary directory, default is `./configs`")
+	return cmd
+}
+
+// NewImportCfgsCmd creates a importCfg command.
+func NewImportCfgsCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "import-configs [--dir directory]",
```

Comment:
https://github.com/pingcap/dm/pull/1746
How about `config export` and `config import`? This command looks not compatible with the newer design.